Tests/MSFT_xIPAddress.Tests.ps1

```diff
@@ -117,7 +139,7 @@ InModuleScope MSFT_xIPAddress {
 
             It 'should be $null' {
                 $Splat = @{
-                    IPAddres = '10.0.0.2'
+                    IPAddress = '10.0.0.2'
                     InterfaceAlias = 'Ethernet'
                 }
                 $Result = ValidateProperties @Splat -Apply
```

Comment:
I would add some calls to make sure the Mocks have been called to prove that its doing more then just running without failure.